App/templates/base.html.twig

```diff
@@ -17,9 +17,11 @@
     </head>
     <body>
         <header class="title">
-            <p>Qui</p>
-            <p>Paie</p>
-            <p>Quoi</p>
+            <a href="{{ path ('groups_home') }}">
+                <p>Qui</p>
+                <p>Paie</p>
+                <p>Quoi</p>
```

Comment:
Semantically speaking I'm not fan with this logo, it is not really three paragraph nor three different entities. Maybe you can achieve something similar by juste remove `p` tags and use CSS to force split the content of the `a` tag at each word 👍

Reply:
Done